grafanalib/tests/test_humio.py

```diff
@@ -0,0 +1,25 @@
+"""Tests for Humio Datasource"""
+
+import grafanalib.core as G
+import grafanalib.humio as H
+from grafanalib import _gen
+from io import StringIO
+
+
+def test_serialization_cloudwatch_metrics_target():
```

Comment:
```suggestion
def test_serialization_humio_metrics_target():
```

Reply:
@JamesGibo Oops, good catch. Just updated the PR and cleaned up some of the comments too.